server/__tests__/index.test.js

```diff
@@ -1,18 +1,17 @@
 
 import request from 'supertest';
-import server from '../src/index';
+import app from '../src/app/app';
 
+const { TEST_PORT = 8888 } = process.env;
```

Comment:
Please import env variables from a common place, check [this](https://stackoverflow.com/a/48828102/10322565) for further info

Reply:
Okay, I will make this change.